spec/api-browser-window-spec.js

```diff
@@ -1686,10 +1686,8 @@ describe('BrowserWindow module', () => {
           done()
         })
         w.loadURL(`file://${path.join(fixtures, 'api', 'native-window-open-iframe.html')}`)
-      })
-      it('loads native addons correctly after reload', (done) => {
-        if (!nativeModulesEnabled) return done()
-
+      });
+      (nativeModulesEnabled ? it : it.skip)('loads native addons correctly after reload', (done) => {
```

Comment:
I'm not sure if this is done like this elsewhere or not but my preferred way of skipping tests dynamically is with

```
it('should so things', function testThings () {
  if (condition) return this.skip()
})
```

Although it requires using `function` instead of arrow function it looks cleaner IMO

Reply:
➕

Reply:
the reason i did it differently is that i saw comments that an `afterEach` hook won't be run if a test is skipped dynamically. I suppose these tests don't have them, but this seems like a more robust method when you know before running any test code whether or not the test will be skipped?